Completion.py

```diff
@@ -111,14 +139,12 @@ def read_settings():
         settings["replace_file_path"] = (replace["from"], replace["to"])
     return settings
 
-
-def is_cpp(view):
-    '''Determine if the given view location is c++ code'''
-    try:
-        return view.match_selector(view.sel()[0].begin(), 'source.c++')
-    except:
-        return False
-
+def lang(view):
+    languages = read_settings()['languages']
```

Comment:
I'm not sure if it's a good idea to parse and load settings every time. The alternative is to cache this, but then we lose the ability to edit the settings file on the fly (do we want this ability? I'm okay without it but you may have other opinions :smile: ). I don't know how to add a hook so that the cache is updated whenever that file is edited.